tests/helpers/runif.py

```diff
@@ -137,8 +137,10 @@ def __new__(
             reasons.append("unimplemented on Windows")
 
         if tpu:
-            conditions.append(not _TPU_AVAILABLE)
+            env_flag = os.getenv("PL_RUN_TPU_TESTS", "0")
```

Comment:
@awaelchli Do we need env variable logic here as well? I would have to export it every time I would have to run a test on TPUs

Reply:
I suppose the check with _TPU_AVAILABLE is enough in our limited scenarios. Would have to also update the usage in conftest.py then

Reply:
I would prefer to use `PL_RUN_TPU_TESTS` even if it is redundant. This would make testing consistent and less ambiguous for contributors.

Reply:
well then let's describe it in the contribution guide because it is not very initiative to see extra env variables for run tests

Reply:
Did we reach a consensus here? I personally prefer not to check for the env variable

Reply:
+1 for not checking it